src/components/footer.js

```diff
@@ -1,35 +1,83 @@
-          fill='white'
-        />
-        <path d='M23.3248 13L32.6497 29H14L23.3248 13Z' fill='white' />
-        <line x1='43.5' y1='2.18557e-08' x2='43.5' y2='44' stroke='#5E5E5E' />
-      </svg>
-    </Link>
-  )
-}
+        <Stack spacing={{ base: '4', md: '5' }}>
+          <Stack justify='space-between' direction='row' align='center'>
+            <Stack
+              as={Link}
+              href={'/'}
+              direction={'row'}
+              alignItems={'center'}
+              spacing={{ base: 2, sm: 4 }}
+            >
+              <Image
+                w={225}
+                src={'/pangeo-forge-text-only-white.png'}
+                alt={'Pangeo-Forge logo'}
+              />
+            </Stack>
 
-const Footer = () => {
-  return (
-    <Box as='footer' sx={{ mt: [6] }}>
-      <Box sx={{ margin: '40px auto', textAlign: 'center' }}>
-        <Vercel />
-      </Box>
+            <ButtonGroup variant='ghost'>
+              <IconButton
+                as={Link}
+                href='https://discourse.pangeo.io/'
+                aria-label='Discourse'
+                icon={<FaDiscourse fontSize='1.85rem' />}
```

Comment:
Especially as you may tweak the theme, it might be helpful to consistently use the theme-configured font sizes (i.e. `xs`...`9xl`) instead of one-off `rem`- or `px`-based definitions.